src/plugins/management/public/management_sections_service.ts

```diff
@@ -17,22 +17,47 @@
  * under the License.
  */
 
-import { ReactElement } from 'react';
 import { ManagementSection, RegisterManagementSectionArgs } from './utils';
-import { managementSections } from './components/management_sections';
+import {
+  IngestSection,
+  DataSection,
+  InsightsAndAlertingSection,
+  SecuritySection,
+  KibanaSection,
+  StackSection,
+} from './components/management_sections';
 
 import {
   ManagementSectionId,
   SectionsServiceSetup,
-  SectionsServiceStart,
   SectionsServiceStartDeps,
+  DefinedSections,
+  ManagementSectionsStartPrivate,
 } from './types';
+import { createGetterSetter } from '../../kibana_utils/public';
+
+const [getSectionsServiceStartPrivate, setSectionsServiceStartPrivate] = createGetterSetter<
```

Comment:
Would you mind explaining this bit?
Why is the getter setter called `private`?

Reply:
Its only used internally.

Reply:
I think all getter-setters are used within the bundle they are defined in. Not sure about this.

Reply:
I'm not sure I understand, is there a change you'd recommend?

Reply:
Just found the "Private" part a bit unclear. Not critical though.